man/man1/pkcsconf.1.in

```diff
@@ -35,16 +35,36 @@ set the SO PIN
 specify the token slot for the operation
 .IP "\fB-U\fP \fIUSERPIN\fP" 10
 the current user pin (for use when changing the user pin; -u and -p options);
-if not specified, user will be prompted
+if not specified, the value in environment variable OCK_USER_PIN is used or
+user will be prompted
 .IP "\fB-S\fP \fISOPIN\fP" 10
 the current Security Officer (SO) pin (for use when changing the SO pin;
--P option); if not specified, user will be prompted
+-P option); if not specified, the value in environment variable OCK_SO_PIN
+is used or user will be prompted
 .IP "\fB-n\fP \fINEWPIN\fP" 10
 the new pin (for use when changing either the user pin or the SO pin; -u, -p
 and -P options); if not specified, user will be prompted
```

Comment:
Maybe explicitly state that the new pin is never obtained from an environment variable.

Reply:
Ok